draft-kohbrok-mimi-portability.md

```diff
@@ -134,12 +138,20 @@ struct {
   MigrationResponse migration_response;
   opaque signature<V>;
 } MigrationInit;
+
+struct {
+  FQDN target_ds_domain;
+} FQDNProposal
 ~~~
 
 The client also sends a Commit message to the group that contains an update to
 the FQDN group context extension. The Commit only contains an FQDNProposal, no
 other proposals are allowed.
 
+TODO: Should the FQDNProposal include the MigrationResponse s.t. other clients
+can validate the process at least to some degree? Also, why not include the
+commit in the MigrationInit message?
```

Comment:
Turns out that no, it shouldn't. Also, no proposal. Instead put the FQDN into the AAD.